metagraph/src/cli/align.cpp

```diff
@@ -350,10 +342,7 @@ int align_to_graph(Config *config) {
     ThreadPool thread_pool(get_num_threads());
     std::mutex print_mutex;
 
-    if (graph->get_mode() == DeBruijnGraph::PRIMARY) {
-        logger->trace("Primary graph wrapped into canonical");
-        graph = std::make_shared<CanonicalDBG>(graph);
-    }
+    graph = wrap_graph(graph);
```

Comment:
bad name

Reply:
How about something like `sanitize_graph`? Any other suggestions?

Reply:
Neither of them is descriptive.

It might be hard to come up with a good name because it has two things.

Just do the canonicalization outside and pass a graph for adding a cache.

Then it can be called `add_cache`. It doesn't matter if this has to re-apply the CanonicalDBG wrapper inside.

Reply:
ok, I split the two functions